[Problem] missing dependencies in Ubuntu Docker

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

0.21 (Development)

### Full version info

```shell
not relevant / unrelated (compiled from source)
```


### Subproject(s) affected?

None

### Problem description

When compiling FreeCAD using the Ubuntu Docker (Dockerfile.Ubuntu) in /tools/build/Docker/ dependencies are missing.

1. qtwebengine5-dev missing = cmake not successful (Qt5WebEngineWidgets)
2. qtwayland5 missing = not possible to run FreeCAD with GUI under Wayland

**Output regarding qtwebengine5-dev // Qt5WebEngineWidgets**
```
CMake Error at /usr/lib/x86_64-linux-gnu/cmake/Qt5/Qt5Config.cmake:28 (find_package):
  Could not find a package configuration file provided by
  "Qt5WebEngineWidgets" with any of the following names:

    Qt5WebEngineWidgetsConfig.cmake
    qt5webenginewidgets-config.cmake

  Add the installation prefix of "Qt5WebEngineWidgets" to CMAKE_PREFIX_PATH
  or set "Qt5WebEngineWidgets_DIR" to a directory containing one of the above
  files.  If "Qt5WebEngineWidgets" provides a separate development package or
  SDK, be sure it has been installed.
Call Stack (most recent call first):
  cMake/FreeCAD_Helpers/SetupQt.cmake:31 (find_package)
  CMakeLists.txt:71 (include)
```
[CMakeOutput__Qt5WebEngineWidgets_error.log](https://github.com/FreeCAD/FreeCAD/files/11941085/CMakeOutput__Qt5WebEngineWidgets_error.log)


**Output regarding qtwayland5**
```
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: eglfs, linuxfb, minimal, minimalegl, offscreen, vnc, xcb.
```

I haven't checked the other docker files. 



### Anything else?

**qtwebengine5-dev missing**

- https://forum.freecad.org/viewtopic.php?t=35356
- https://stackoverflow.com/questions/61425552/how-to-install-use-qtwebengine-widgets-on-linux


### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

--- a/tools/build/Docker/ubuntu.sh
+++ b/tools/build/Docker/ubuntu.sh
@@ -12,4 +12,5 @@ apt-get install --no-install-recommends --yes build-essential cmake doxygen git 
     libvtk-dicom-dev libx11-dev libxerces-c-dev libxmu-dev libxmuu-dev \
     libzipios++-dev netgen netgen-headers pyside2-tools python3-dev \
     python3-matplotlib python3-pivy python3-ply python3-pyside2.qtsvg \
-    python3-pyside2.qtuitools qtchooser qttools5-dev shiboken2 swig
+    python3-pyside2.qtuitools qtchooser qttools5-dev shiboken2 swig \
+    qtwebengine5-dev qtwayland5

